src/routes/profile.js

```diff
@@ -191,7 +174,12 @@ router.get(
         if (err) {
           console.log(err);
         } else {
-          res.render('profile/changeavatar', { userData: foundUser });
+          res.render('profile/changeavatar', {
+            userData: foundUser,
```

Comment:
Let's clean this up a bit (refactor) to not pass in the whole user in. Safer code.